FeedStoreChallenge/MemoryFeedStore.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MemoryFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Damian Modernell on 18/3/2564 BE.
+//  Copyright © 2564 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public final class MemoryFeedStore: FeedStore {
+	
+	private var cache: (feed: [LocalFeedImage],timestamp:  Date)?
```

Comment:
Fix spacing to be consistent with the rest of the code:

```suggestion
	private var cache: (feed: [LocalFeedImage], timestamp: Date)?
```